src/xdppcw/inc/xdppcw.h

```diff
@@ -31,6 +31,7 @@ typedef struct _XDP_PCW_LWF_RX_QUEUE {
     UINT64 ForwardingFailuresRscInvalidHeaders;
     UINT64 ForwardingNbsRequested;
     UINT64 ForwardingNbsSent;
+    UINT64 NdisLoopbackPackets;
```

Comment:
```suggestion
    UINT64 LoopbackNblsSkipped;
```

Reply:
In general, XDP deals with "frames" and "buffers", and the generic module also deals with "NBLs" and "NBs". We use "packet" to mean specifically L3 packets.